dev/create-release/release-build.sh

```diff
@@ -322,18 +322,18 @@ if [[ "$1" == "package" ]]; then
   #   'python/pyspark/install.py' and 'python/docs/source/getting_started/install.rst'
   #   if you're changing them.
   declare -A BINARY_PKGS_ARGS
-  BINARY_PKGS_ARGS["hadoop3.2"]="-Phadoop-3.2 $HIVE_PROFILES"
+  BINARY_PKGS_ARGS["hadoop3.3"]="-Phadoop-3 $HIVE_PROFILES"
```

Comment:
Why is this `hadoop3.3` while the profile is `hadoop-3`?

Reply:
Keep the package name as hadoop3.3 to let user know the download version is hadoop3.3.